data/human/post-war reactions.txt

```diff
@@ -0,0 +1,260 @@
+				`	"Oh, the Pug were real alright. I met some and fought them."`
+					goto nobody
+				`	"I think you both need to shave."`
+					goto scowl
+			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`
+			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru, he keeps ranting against this Navy guy, uh, this Admiral..." He snaps his fingers trying to remember the name.`
+			`	"Admiral Danforth," says the right twin.`
+			`	"Right. Danforth. This Councilor wants Danforth to defend against 'unknown dangers at the fringes of human space'. Don't we have enough immediate problems without imagining new horrors?"`
+			`	"Like terrorists nuking us," says the right twin.`
+			choice
+				`	"The Navy is skilled and honorable. I trust them to protect us."`
+					goto eyes
+				`	"No, don't worry about that. I ended the bomb threat."`
+				`	"Maybe you're right."`
+					goto shrug
+				`	"All we can do is get on with our lives, right?"`
+					goto shrug
+				`	"If you don't shave soon, people will think you're wearing masks."`
+					goto scowl
+			label nobody
+			`	Both of them give hard belly laughs, sending waves through their mustaches from the breeze of their breath.`
+			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
+			choice
+				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
+				`	"Never mind."`
+					goto shrug
+				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
+					goto scowl
+			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`
+			`	The right one says apologetically, "Thank you for your service, Captain <last>."`
+			`	They both give clumsy but well-intentioned salutes and return to their ships.`
+				decline
+			label eyes
+			`	The left twin laughs and says, "Pardon us if we keep our eyes open and trust to our own guns." Both twins nod and depart to their ships.`
+				decline
+			label shrug
+			`	Both twins shrug and leave for their ships.`
+				decline
+			label scowl
+			`	Both twins' faces twist in a way that might indicate embarrassment or annoyance. It's hard to tell since their facial hair is halting any emotive power. They walk away in the direction of their ships.`
+				decline
+
+
+mission "War Reactions: Navy and Free Worlds Captains"
+	landing
+	minor
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "spaceport"
+		# This distance from Sirius puts them a few jumps from the edge of human space, or anywhere in the Free Worlds
+		near "Sirius" 5 100
```

Comment:
It also seems to include much of the Deep. While those systems are near the edge of human space, it's not really in the same way as the systems, for example, in the far north are.

Testing this filter with relevant changes (this also includes all the changes from HR and the Wanderer story because I can't be bothered finding a save file with only the changes from the FW story right now):

<details> <summary>Planets</summary>

- Albatross
- Alexandria
- Alfheim
- Amazon
- Antipode
- Arabia
- Asgard
- Big Sky
- Bivrost
- Bloodsea
- Bluestone
- Bounty
- Bourne
- Burthen
- Calda
- Canyon
- Charon Station
- Clark
- Cornucopia
- Crossroads
- Dancer
- Darkstone
- Deep
- Dune
- Farpoint
- Featherweight
- Fenrir Station
- Flood
- Foundry
- Gemstone
- Glaze
- Glory
- Harmony
- Haze
- Heartland
- Helheim
- Hippocrates
- Hopper
- Icefall
- Ingot
- Leviathan Station
- Lichen
- Longjump
- Maelstrom
- Mani
- Memory
- Mere
- Midgard
- Moonshake
- Muninn Station
- Muspel
- New Argentina
- New Britain
- New Greenland
- New Holland
- New Iceland
- New India
- New Kansas
- New Portland
- New Tibet
- New Wales
- New Washington
- Nifel
- Nimbus
- Norn
- Oasis
- Oblivion
- Pearl
- Placer
- Poisonwood
- Poseidos
- Prime
- Quicksilver
- Rand
- Rust
- Serpens
- Shangri-La
- Shorebreak
- Shroud
- Sinter
- Skillet
- Skymoot
- Solace
- Splashdown
- Starcross
- Sundive
- Sundrinker
- Sunracer
- Thunder
- Trinket
- Triton Station
- Trove
- Twinstar
- Vail
- Valhalla
- Wayfarer
- Windblain
- Winter
- Wyvern Station
- Zug

</details>

Reply:
The Deep is intentionally part of the filter. I'm selecting systems that have not been touched by the main campaign conflict. Quicksilver should not have been on that list, so I do need to update it.

Reply:
This save has done the main campaign without ever leaving human space: 
[False Facsimile~3018-12-23 main campaign done.txt](https://github.com/endless-sky/endless-sky/files/11244725/False.Facsimile.3018-12-23.main.campaign.done.txt)